### DISCUSSION TITLE
Accuracy not improving

### DISCUSSION THREAD

**Discussion by jhaggle** [open-mmlab/mmsegmentation#2244] [General]
I am following this tutorial:
https://github.com/open-mmlab/mmsegmentation/blob/master/demo/MMSegmentation_Tutorial.ipynb

After one quick run with `cfg.runner.max_iters = 200` I get Accuray 99.14% for the background, 93% for class A and 94 % for class B.
When I increase number of iterations to `cfg.runner.max_iters = 4000` the result hardly improves. Why does the result not improve? What can I do to improve the result except for using more annotated images in the training?

**Comment by MengzhangLI** [comment]
Hi, if you work on your own dataset, could you try larger maximum itertaions?

From config files you can see MMSegmentation uses large iteration numbers during training, from [20k to 320k](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/_base_/schedules). Although our commonly used iteration number is 40k, 80k and 160k, they are all much larger than 2k or 4k which you use in your dataset.

Looking forward to your reply!

**Reply by jhaggle** [reply]
Thanks for the reply! Increasing to 40000k made it better. Using more annotated images even more.

**Comment by MeowZheng** [comment]
Improving the model performance is the final aim of all deep learning researcher .

You might try to use a stronger model or stronger data augmentation or an advanced training schedule.

I have converted this issue to discussion as it is an open discussion topic.
